hyukju/app.js

```diff
@@ -41,6 +41,134 @@ app.post("/users", async (req, res, next) => {
   res.status(201).json({ message: "successfully created" });
 });
 
+app.post("/posts", async (req, res, next) => {
```

Comment:
```suggestion
app.post("/posts", async (req, res) => {
```
미들웨어와 관련된 next 인자는 지금 필요하지 않을 것 같습니다.